VmAgent.Core/VmAgent.Core.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <PackageVersion>1.1.10</PackageVersion>
+    <PackageVersion>1.2.0</PackageVersion>
```

Comment:
why not 1.1.11? :)

Reply:
This change should technically be a major version bump--i removed a method from a public-facing interface, which is a breaking change. I opted for bumping the minor version instead though because it was a method that wasn't in use yet.
I'd prefer bumping the major version over bumping the patch version, though, because changing the patch version implies that there is no change to the public API. We should only be bumping the patch version if it's an implementation change that doesn't affect compilation.

Reply:
fair enough, though our internal code is the only consumer of these APIs. LGTM!